### PR TITLE
Simplify VSync toggling

### PR DESCRIPTION
**Pull request by JNechaevsky**
This do two things:

1. Simplifies VSync toggling a bit.
2. My "idiocy" fix seems to be no longer working on SDL 2.26.4 / Windows 11 Pro with latest updates, and causing screen freeze again (same applies to Crispy-Doom and CRL). This must be good, so we no longer need to rely on such odd condition.

Despite of this toggle works fine on my end, some extra testing from anyone who is using Windows 11 will be much, much appreciated!

### CODE REVIEW

**Comment by rfomin** [comment]
Works well on Windows 10. Thank you!

**Comment by JNechaevsky** [comment]
One more important thing (thanks @rfomin for pointing out): despite of hires widescreen rendering are also invoking I_ResetScreen function, no freeze is happening. Why? I wish I know. Can only assume that something is going wrong by toggling vsync by non-SDL function and then recreating texture. :thinking:

File: src/i_video.c [hunk 1]
```diff
@@ -1560,16 +1565,6 @@ static void I_InitGraphicsMode(void)
 
    I_GetWindowPosition(&window_x, &window_y, v_w, v_h);
 
-#ifdef _WIN32
-   // [JN] Windows 11 idiocy. Indicate that window using OpenGL mode (while it's
-   // a Direct3D in fact), so SDL texture will not be freezed upon vsync
-   // toggling.
-   if (I_CheckWindows11())
```
Comment (fabiangreffrath) [inline src/i_video.c:1567]: Can we remove this function entirely?

Comment (rfomin) [reply src/i_video.c:1567]: Done in https://github.com/fabiangreffrath/woof/pull/977/commits/01e685758f0babfc1d4bdd25958de1e4a86d0c15 Is it OK to merge? It works surprisingly well, and in exclusive full screen mode too.

**Review by fabiangreffrath — APPROVED** [review]
Thank you guys!

**Pull request merged**